wikibase/1.30/bundle/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu as fetcher
+
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends unzip
+
+ADD https://extdist.wmflabs.org/dist/extensions/OAuth-REL1_30-352260e.tar.gz .
+
+# Creates /mediawiki-extensions-Wikibase-REL1_30
+RUN tar xzf OAuth-REL1_30-352260e.tar.gz && rm *.tar.gz
+
+
+FROM wikibase/wikibase:1.30
```

Comment:
I'm not sure how docker cloud and this build dependency will actually work out, might be worth doing some investigation.

AFAIK this bundle image and the base image will likely get built simultaneously and thus updates to the base image wouldn't be reflected in the bundle image on this build :/